src/typescript/azure-openapi-validator/rules/EnumUniqueValue.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from "../rule";
+export const EnumUniqueValue: string = "EnumUniqueValue";
+
+rules.push({
+  id: "R3024",
+  name: EnumUniqueValue,
+  severity: "warning",
+  category: "ARMViolation",
+  mergeState: MergeStates.individual,
+  openapiType: OpenApiTypes.arm | OpenApiTypes.dataplane,
+  appliesTo_JsonQuery: "$..*[?(@.enum)]",
+  run: function*(doc, node, path) {
+    const msg: string = `Enum must not contain case-insensitive duplicated value and make sure every value in enum unique.`;
```

Comment:
could you update the message to contain enum value?

Reply:
Updated.